packages/discord.js/src/structures/interfaces/TextBasedChannel.js

```diff
@@ -78,8 +78,10 @@ class TextBasedChannel {
    * (see {@link https://discord.com/developers/docs/resources/message#allowed-mentions-object here} for more details)
    * @property {Array<(AttachmentBuilder|Attachment|AttachmentPayload|BufferResolvable)>} [files]
    * The files to send with the message.
-   * @property {Array<(ActionRowBuilder|ActionRow|APIActionRowComponent)>} [components]
-   * Action rows containing interactive components for the message (buttons, select menus)
+   * @property {Array<(ActionRowBuilder|MessageTopLevelComponent|APIMessageTopLevelComponent)>} [components]
+   * Action rows containing interactive components for the message (buttons, select menus) and other
+   * top level components <info>When using components v2 the flag `MessageFlags.IsComponentsV2` needs to be set and
```

Comment:
```suggestion
   * top level components <info>When using components v2 the flag {@link MessageFlags.IsComponentsV2} needs to be set and
```

Should probably use `@link` tag as that would be better for the docs website